Themes/default/css/index.css

```diff
@@ -1409,10 +1430,22 @@ ul li.greeting {
 
 /* The jump to box */
 #display_jump_to {
-	padding: 11px 0 5px 0;
+	padding: 5px 0;
+	margin: 4px 0;
 	float: right;
 	font-size: 1em;
 }
+#topic_icons #message_index_jump_to {
+	padding: 0;
+	float: right;
+}
+
+/* The category separator string is 29 dashes */
+#display_jump_to_select,
+#message_index_jump_to_select,
+#quick_mod_jump_to_select {
+	width: 29ch;
+}
```

Comment:
not sure how I feel about this.
I discussed about this months ago on discord, but I'm not happy with either behavior, and with your changes we could end up with this:
![Screenshot_31](https://user-images.githubusercontent.com/3318127/150698407-d7566b8c-1f6f-42a6-bc51-2af523cb00ec.png)

Not sure I like it, I'm not opposed to it, but I think it'd bring more questions than the previous behavior.

Reply:
Interesting. What browser is that screenshot from? I don't see that in anything I try it in.

Reply:
Firefox using the responsive view

Reply:
I guess that's just Firefox's algorithm for deciding where to place the expanded select menu. It's not overflowing the window even with that long item, though, so it seems more like an aesthetic issue that a functional one. If we have to choose between that or the menu suddenly change size when the user mouses over it, the latter is more important to solve.

Reply:
I checked this again and you forgot to add `#search_jump_to_select`

Reply:
@Sesquipedalian

Reply:
Ah yes. Thanks!

Reply:
Fixed